### PR TITLE
slides/01-introduction: update

### PR DESCRIPTION
**Pull request by fjmolinas**
- update images
- replace algramo usecase by locha mesh

### CODE REVIEW

File: slides/01-introduction/introduction.md [hunk 1]
```diff
@@ -455,12 +448,13 @@ class: center, middle
...
+## IoT projects using RIOT
 
 .center[
-**Algramo: Automatic cereal dispenser**<br><br>
-<img src="images/almagro-feeder.png" alt="" style="width:200px"/><br/>
-<a href="https://www.indiegogo.com/projects/the-venture-algramo-chile#/">More information</a>
+**Locha Mesh: A Decentralized Mesh Network**<br><br><br><br>
+<img src="images/turpial-finished-board.jpg" alt="" style="width:200px"/>
+<img src="images/LogotipoTurpial-Color.20-09-19.svg" alt="" style="width:300px"/><br/>
+<a href="https://locha.io/">More information</a>
```
Comment (emmanuelsearch) [inline slides/01-introduction/introduction.md]: For Continental we could add the [photo](https://www.continental-mobility-services.com/media/1654/accm_access_telematics_description_image1_552x311.png) of their RIOT-based product?

Comment (emmanuelsearch) [reply slides/01-introduction/introduction.md]: Actually, maybe we should put products first (before deployments & projects). What do you think?

Comment (fjmolinas) [reply slides/01-introduction/introduction.md]: Did that

**Comment by fjmolinas** [comment]
TODO: mention UWB and linux image is a bit off.

**Review by emmanuelsearch — APPROVED** [review]

**Pull request merged**